### ISSUE TITLE
directory "mrjob" masks Python library "mrjob"

### ISSUE THREAD

**Issue by SamPenrose** [mozilla-metrics/fhr-toolbox#10]
I'd like to rename it ... OK w/ you @bsmedberg ? Others? I propose "fhrpy".

**Comment by bsmedberg** [comment]
Yeah, although a longer name like "python-analysis" might be better.

**Comment by SamPenrose** [comment]
I don't care if we exactly conform to PEP8:
  https://www.python.org/dev/peps/pep-0008/#package-and-module-names
 but it's going to be a package name, so it can't have "-". I'll go with "pythonanalysis".

**Comment by SamPenrose** [comment]
Over the weekend I had a painful introduction to how Hadoop (doesn't) play with Python's packaging system; so I'm going to research a bit more. It would be good to have One Correct Way to make PYTHONPATH and Hadoop play nicely together.
